dns-controller/README.md

```diff
@@ -1,22 +1,29 @@
-dns-controller creates DNS records
+# dns-controller
 
-In the bring-up of a new cluster, protokube has already ensured that we have an etcd cluster and an apiserver.  It also
-sets up DNS records for the etcd nodes (this is a much simpler problem, because we have a 1:1 mapping from an etcd
-node to a DNS name.)
+dns-controller creates DNS records.
 
-However, none of the nodes can reach the api server to register.  Nor can end-users reach the API.  In future
-we might expose the API server as a normal service via Type=LoadBalancer or via a normal Ingress, but for now
-we just expose it via DNS.
+In the bring-up of a new cluster, `protokube` has already ensured that 
+we have an `etcd` cluster and an `apiserver`.  It also sets up DNS 
+records for the `etcd` nodes (this is a much simpler problem, because 
+we have a 1:1 mapping from an `etcd` node to a DNS name.)
 
-The dns-controller recognizes annotations on nodes.
+However, none of the nodes can reach the api server to register.  Nor 
+can end-users reach the API.  In future we might expose the API server 
+as a normal service via `Type=LoadBalancer` or via a normal Ingress, 
+but for now we just expose it via DNS.
 
-`dns.alpha.kubernetes.io/external` will set up records for accessing the resource externally
+The dns-controller recognizes annotations on nodes.
```

Comment:
I'm unsure either tbh, I only slightly improved the formatting in this file, I can't vouch for its accuracy.